ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -718,12 +718,12 @@ def getStateTime(self, status):
             splitLine = l.split()
             if checkstr == splitLine[0]:
                 year = datetime.datetime.now().year
-                if datetime.datetime.strptime(str(datetime.datetime.now().year)+'/'+splitLine[2]+' '+splitLine[3], "%Y/%m/%d %H:%M:%S") > datetime.datetime.now():
+                if datetime.datetime.strptime(splitLine[2]+' '+splitLine[3], "%Y-%m-%d %H:%M:%S") > datetime.datetime.now():
                     year = year - 1
-                timestr = str(year)+'/'+splitLine[2]+' '+splitLine[3]
+                timestr = splitLine[2]+' '+splitLine[3]
```

Comment:
As above. Might be useful to change the old notation to have a "-" and not a "/". That will mean you just have to check if there is a year in the string or not.

Reply:
See my reply above, the changes are in a new commit